### DISCUSSION TITLE
Compilation error.

### DISCUSSION THREAD

**Discussion by senkuma** [azerothcore/azerothcore-wotlk#9600] [General]
Heyo! I have everything downloaded like I should, I compiled a trinitycore before with no errors but I thought I would use AzerothCore instead because well, I like it more.

But I get errors when compiling. I had no errors at all when using cmake and so on. But I don't really know what to do with this error.

![acdelta](https://user-images.githubusercontent.com/20373639/145316652-23065efe-dcbe-400e-9a2e-867c8f1c5dab.JPG)
.

**Comment by Nyeriah** [comment]
What version of OpenSSL are you using? Versions higher than 1.1 are not supported.

https://www.azerothcore.org/wiki/windows-requirements

**Reply by senkuma** [reply]
Yeah that worked, it seems like I had a little older version of it, but as soon as I installed 1.1 it worked perfectly. Thanks :)

**Reply by senkuma** [reply]
My auth and worldserver is instantly closing when trying to open them now. It says I need libmariadb.dll. When I add it they either instantly close or when I add another version of the dll it says 0xc000007b error and closes.

I've had an issue like that before a long time ago but I really can't remember what the problem is.

**Reply by Nyeriah** [reply]
I don't use mariadb but usually you must copy the libraries contained in your SQL installation folder. If you download them from elsewhere they'll usually have version/architecture incompatibilities that will prevent the server from running.

**Reply by senkuma** [reply]
I don't either so I found it weird that I needed it. However I didn't choose 64x when using cmake, now I did and I recompiled and such. So now I don't need mariadb, but the servers just instantly close, not saying anything, basically just blinking and then gone. Very weird.

**Reply by senkuma** [reply]
It is solved.

I was just being stupid, I copied the config files from the config folder straight into the RelWithDebInfo map, when I should have left them inside the config folder. I did it out of habit from TrintityCore.
